q2_types/feature_table/_transformer.py

```diff
@@ -111,3 +111,14 @@ def _6(data: biom.Table) -> BIOMV210Format:
 @plugin.register_transformer
 def _7(data: biom.Table) -> pd.DataFrame:
     return _table_to_dataframe(data)
+
+
+@plugin.register_transformer
+def _8(ff: BIOMV100Format) -> BIOMV210Format:
+    data = _parse_biom_table_v100(ff)
+    data = _drop_axis_metadata(data)
```

Comment:
Just noticed that the `_parse_biom_table_v100` already applies `_drop_axis_metadata`, so that step wasn't necessary. I added the second utility function you requested though, good idea.